### ISSUE TITLE
Xournal++ on e-ink tablets

### ISSUE THREAD

**Issue by utcafj** [xournalpp/xournalpp#4677]
Hi! I'm using your notetaking software on a Pinenote e-ink tablet for my studies. I've upgraded from the latest release to the master version, which really makes things faster. Nevertheless, I noticed a few things:

1. The erasor leaves some traces. Can I make it completely white somehow?
2. Sometimes drawing is a bit laggy. Any idea why this is the case? I can try to take a video to highlight the issue
3. Opening the preferences menu is much slower than on my laptop. It takes even a couple of seconds

**Describe alternatives you've considered**
I tried some other software (xournal and rnote), but I like xournal++ better :D

**Comment by rolandlo** [comment]
1. The eraser fix from #4659 was merged just 10 hours ago. I guess, you haven't had it applied?! Or is this another issue? Are you talking about the standard eraser (or delete stroke or whiteout eraser)?
2. One reason may be  #4662 . Some unrelated optimizations are made in  #4637 by @hrdl-github (who tests on the Pinenote, too).
3. I have observed that on the PinePhone as well. We could try to find the cause.
